src/Yoti.Auth/YotiAttribute.cs

```diff
@@ -13,25 +15,40 @@ namespace Yoti.Auth
     /// </summary>
     public class YotiAttribute<T> : BaseAttribute
     {
-        internal readonly YotiAttributeValue Value;
+        private readonly byte[] _data;
+        private readonly AttrpubapiV1.ContentType _type;
 
-        public YotiAttribute(string name, YotiAttributeValue value) : base(name, value)
+        public YotiAttribute(string name, AttrpubapiV1.ContentType type, byte[] data) : base(name)
         {
-            Value = value;
+            _type = type;
+            _data = data;
         }
 
-        public YotiAttribute(string name, YotiAttributeValue value, List<Anchor> anchors) : base(name, value, anchors)
+        public YotiAttribute(string name, AttrpubapiV1.ContentType type, byte[] data, List<Anchor> anchors) : base(name, anchors)
         {
-            Value = value;
+            _type = type;
+            _data = data;
+        }
+
+        public byte[] Data()
+        {
+            return _data;
+        }
+
+        public AttrpubapiV1.ContentType Type()
```

Comment:
This looks like we are exposing ContentType from the protobuf....  we must hide that away.  No protobuf stuff should ever be exposed.

And I don't think we need to have an additional _Type_ on _Attribute_ at all.  We already have generic types for _Attribute_ that determine the type of the underlying value.